content/sensu-go/5.11/installation/install-sensu.md

```diff
@@ -5,385 +5,330 @@ description: "Sensu Go is available for Linux, Windows (agent and CLI only), mac
-
-{{< platformBlock "RHEL/CentOS" >}}
-
-#### RHEL/CentOS
-
-Add the Sensu repository.
+{{< highlight "RHEL/CentOS" >}}
+# Add the Sensu repository
+curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
 
-{{< highlight shell >}}
-curl -s https://packagecloud.io/install/repositories/sensu/stable/script.rpm.sh | sudo bash
+# Install the sensu-go-backend package
+sudo yum install sensu-go-agent
 {{< /highlight >}}
 
-Install the `sensu-go-cli` package.
+{{< highlight "Windows" >}}
+# Download the Sensu agent for Windows amd64
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.0/sensu-go-agent_5.11.0.4171_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.11.0.4171_en-US.x64.msi"
+# Or for Windows 386
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.0/sensu-go-agent_5.11.0_2380_en-US.x86.msi  -OutFile "$env:userprofile\sensu-go-agent_5.11.0.4171_en-US.x86.msi"
 
-{{< highlight shell >}}
-sudo yum install sensu-go-cli
+# Start the installation wizard
+msiexec.exe /i $env:userprofile\sensu-go-agent_5.11.0.4171_en-US.x64.msi
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
+{{< /language-toggle >}}
 
-{{< platformBlock "Windows" >}}
+##### 2. Configure and start {#agent-start}
 
-#### Windows
+You can configure the Sensu agent using `sensu-agent start` flags or an /etc/sensu/agent.yml file, the former taking precedence.
+At a minimum, the Sensu agent requires the `--backend-url` flag, but here are some other useful configs and templates.
 
-Download sensuctl for Windows `amd64`.
+{{< language-toggle >}}
 
-{{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-enterprise-go_5.11.1_windows_amd64.zip  -OutFile C:\Users\Administrator\sensu-enterprise-go_5.11.1_windows_amd64.zip
+{{< highlight Docker >}}
+# If you are running the agent locally on the same system as the Sensu backend, add `--link sensu-backend` to your Docker arguments and change the backend URL to `--backend-url ws://sensu-backend:8081`.
+docker run -v /var/lib/sensu:/var/lib/sensu -d --name sensu-agent sensu/sensu:latest sensu-agent start --backend-url ws://sensu.yourdomain.com:8081 --subscriptions webserver,system --cache-dir /var/lib/sensu
 {{< /highlight >}}
 
-See the guide to [binary-only distributions][12] to download sensuctl for Windows `386` and verify your download using checksums.
-
-{{< platformBlockClose >}}
-
-{{< platformBlock "macOS" >}}
-
-#### macOS
-
-Download the latest release.
-
-{{< highlight shell >}}
-curl -LO https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-enterprise-go_5.11.1_darwin_amd64.tar.gz
+{{< highlight "Docker Compose" >}}
+---
+version: "3"
+services:
+  sensu-agent:
+    image: sensu/sensu:latest
+    ports:
+    - 3031:3031
+    command: "sensu-agent start --backend-url ws://sensu-backend:8081 --log-level debug --subscriptions webserver --api-host 0.0.0.0"
 {{< /highlight >}}
```

Comment:
To do:

- [ ] Make sure `docker run` example is correct and reflects best practices
- [ ] Make sure Docker Compose example is consistent with `docker run` example

Reply:
LGTM 👍